docs/react-translation-log.html

```diff
@@ -0,0 +1,11456 @@
+<!DOCTYPE html>
```

Comment:
I think that it only makes sense to keep the `react-translation-log.html` file in context of the version that was used as the source, e.g.:

- `docs/next/react-translation-log.html`
- `docs/12.1/react-translation-log.html`
 
Otherwise the files will wrongly overwrite each other.

Reply:
You're right, but this will require adding some logic to split the logs between versions, when the entire `docs:build` is called.

Reply:
Please do it, because otherwise we will get problematic, unmeaningful changes or conflicts in this file. It might be in a separate PR if you prefer.